website/www/site/content/en/get-started/from-spark.md

```diff
@@ -0,0 +1,245 @@
+    <td><code>$ python beam_pipeline.py</code></td>
+</tr>
+</table>
+{{< /table >}}
+
+## Transforms
+
+Here are the equivalents of some common transforms in both PySpark and Beam.
+
+{{< table >}}
+|                   | PySpark                               | Beam                                                    |
+|-------------------|---------------------------------------|---------------------------------------------------------|
+| **Map**           | `values.map(lambda x: x * 2)`         | `values | beam.Map(lambda x: x * 2)`                    |
+| **Filter**        | `values.filter(lambda x: x % 2 == 0)` | `values | beam.Filter(lambda x: x % 2 == 0)`            |
+| **FlatMap**       | `values.flatMap(lambda x: range(x))`  | `values | beam.FlatMap(lambda x: range(x))`             |
+| **Group by key**  | `pairs.groupByKey()`                  | `pairs | beam.GroupByKey()`                             |
+| **Reduce**        | `values.reduce(lambda x, y: x+y)`     | `values | beam.CombineGlobally(sum)`                    |
+| **Reduce by key** | `pairs.reduceByKey(lambda x, y: x+y)` | `pairs | beam.CombinePerKey(sum)`                       |
+| **Distinct**      | `values.distinct()`                   | `values | beam.Distinct()`                              |
+| **Count**         | `values.count()`                      | `values | beam.combiners.Count.Globally()`              |
+| **Count by key**  | `pairs.countByKey()`                  | `pairs | beam.combiners.Count.PerKey()`                 |
+| **Take smallest** | `values.takeOrdered(3)`               | `values | beam.combiners.Top.Smallest(3)`               |
+| **Take largest**  | `values.takeOrdered(3, lambda x: -x)` | `values | beam.combiners.Top.Largest(3)`                |
+| **Random sample** | `values.takeSample(False, 3)`         | `values | beam.combiners.Sample.FixedSizeGlobally(3)`   |
+| **Union**         | `values.union(otherValues)`           | `(values, otherValues) | beam.Flatten()`                |
+| **Co-group**      | `pairs.cogroup(otherPairs)`           | `{'Xs': pairs, 'Ys': otherPairs} | beam.CoGroupByKey()` |
+{{< /table >}}
+
+> ℹ️ To learn more about the transforms available in Beam, check the
+> [Python transform gallery](/documentation/transforms/python/overview).
+
+## Using calculated values
+
+Since we are working in potentially distributed environments,
+we can't guarantee that the results we've calculated are available at any given machine.
+
+In PySpark, we can get a result from a collection of elements (RDD) by using
+`data.collect()`, or other aggregations such as `reduce()`, `count()` and more.
+
+Here's an example to scale numbers into a range between zero and one.
+
+{{< highlight py >}}
+import pyspark
+
+with pyspark.SparkContext() as sc:
+    values = sc.parallelize([1, 2, 3, 4])
+    total = values.reduce(lambda x, y: x + y)
+
+    # We can simply use `total` since it's already a Python value from `reduce`.
```

Comment:
Nit: clarify what you mean here by "Python value" (in this case it's an `int` value).

Reply:
Done